.papr.yml

```diff
@@ -45,18 +45,18 @@ context: "CAH smoketested"
 
 inherit: true
 host:
-    distro: fedora/27/cloud/pungi
```

Comment:
This is failing because there's no cloud pungi images. Try just `fedora/28/cloud` instead.